doc_src/index_template.html

```diff
@@ -11,15 +11,14 @@
 
 <h1>OmegaT User Manual</h1>
 
-<table>
+<table width="20%">
 <% languages.each { lang -> %>
     <tr>
         <td>
-            <p>${lang.name}:</p>
+            <p><a href="${lang.code}/index.html">${lang.name}</a></p>
         </td>
-        <td><a href="${lang.code}/index.html">HTML/Web</a></td>
-        <td><a href="https://sourceforge.net/projects/omegat/files/OmegaT%20-%20Standard/OmegaT%20${lang.version}/OmegaT_documentation_${lang.code}.PDF/download">PDF</a></td>
         <td>(<span class="${lang.status}">${lang.version}</span>)</td>
+        <td><a target="blank" href="https://sourceforge.net/projects/omegat/files/OmegaT%20-%20Standard/OmegaT%20${lang.version}/OmegaT_documentation_${lang.code}.PDF/download">PDF</a></td>
```

Comment:
Not sure if I'm missing something but just to be sure we're on the same page, my understanding is that this file builds [this page](https://omegat.sourceforge.io/manual-standard/) and the page that is currently online is not the version we're working on here as it has been edited recently and not built and uploaded since.

The version in the repo turned it into four cells that looked like that: 
-The language name (in plain text) |
-"HTML/Web" as a link to the HTML manual on omegat.sourceforge.io |
-"PDF" as a link to the pdf in the downloads on sourceforge |
-a version and status indicator

eg: French: | HTML/Web | PDF | (3.6.0)

This change would instead display:
-The language as a link to the HTML manual on omegat.sourceforge.io |
-a version and status indicator |
-"PDF" as a link to the pdf in the downloads on sourceforge

eg: French | (3.6.0) | PDF

If so, I think merging the language name and the link to the web version back into one cell makes things confusing now that there are two links:
Is the PDF not translated? And if it is, then where does the link that says French go? Why is there a version number in the middle?

I'm not happy with "Web/HTML" either (and to be honest, it was more of a question, offering two options) but I think we need to somehow indicate "this is the HTML version (hosted on this website)" and "this is the PDF version as a download". 
We could maybe add a header before line 15, indicating what each column contains?

Reply:
We have an old problem with the various OmegaT versions and the way the manual is handled.

- The "standard" version comes with a very outdated "updated" manual.
   https://omegat.sourceforge.io/manual-standard/
- The "latest" version is recent code, but not documented.
   https://omegat.sourceforge.io/manual-latest/

The standard version manuals have not been built in a long time, which explains the discrepancy that you see online. That is ugly.

What we are doing now is **stop** having two versions. We'll always have a version that is properly documented and will organize so that the documentation is produced on time.

That's the reason why it's taking so much time to release this 5.8/6.0 (if you're not sure about the difference: they are the same but 5.8 will be based on Java 8 and 6.0 will be based on Java 11, only 6.0 will be advertised and 5.8 will only be mentioned for people who have _really old machines/OSs_).

That being said, the build process currently builds the "latest" version:
https://omegat.sourceforge.io/manual-latest/

The "Web/HTML" part is kind of unnecessary because this file is also the one that is included in the distribution and is displayed in your browser when you open the help menu in OmegaT. So, people kind of know that the underlined part is a link and will click on their language to have access to their localized manual.

Also, I'm not sure that we need to link to the PDF there. It would be better to keep the links to the PDFs for the omegat.org site, especially since we don't have yet a satisfactory PDF output due to plenty of reasons (check https://sourceforge.net/p/omegat/mailman/omegat-development/thread/4DEF6BE8-6B5E-4938-BCC1-8B16AC7120A8%40traduction-libre.org/#msg37751519).

The manual web hosting is a convenience and people who use it will (most probably) go/refer straight to the localized version that they need and not to the index. So that online index is not as important as its local clone, which is the reason why I prefer to keep its layout and the info it displays super simple.